<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>484</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Register{
border-radius:10px;
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>484</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="rg">
    <widget class="QPushButton" name="btn_MinR">
     <property name="geometry">
      <rect>
       <x>823</x>
       <y>17</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_MinR
{
     border-image:url(:/image/Min01.png);
}
QPushButton:hover#btn_MinR
{
     border-image:url(:/image/Min_hover.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="MyLineEdit" name="lineEdit_Username">
     <property name="geometry">
      <rect>
       <x>518</x>
       <y>312</y>
       <width>317</width>
       <height>45</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	font-size:20px;
}
QLineEdit#lineEdit_Username{
border:1px solid rgb(211,211,211);
}
QLineEdit:hover#lineEdit_Username{
border:1px solid rgb(255,170,0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_star3">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>287</y>
       <width>6</width>
       <height>6</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(251, 173, 88);</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_picLoveR">
     <property name="geometry">
      <rect>
       <x>36</x>
       <y>26</y>
       <width>220</width>
       <height>55</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>55</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/life_ring2.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2R">
     <property name="geometry">
      <rect>
       <x>518</x>
       <y>90</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:27px;
color: rgb(110, 110, 110);</string>
     </property>
     <property name="text">
      <string>用户注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_star2">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>244</y>
       <width>6</width>
       <height>6</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(251, 173, 88);</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_RegisterR">
     <property name="geometry">
      <rect>
       <x>523</x>
       <y>389</y>
       <width>317</width>
       <height>48</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	font-size:24px;
}
QPushButton:hover#btn_RegisterR{
background-color:#A9E2F3;
color: rgb(255, 245, 237);
border-radius:5px;
}
QPushButton#btn_RegisterR{
background-color:#58ACFA;
color: rgb(255, 245, 237);
border-radius:5px;
}




</string>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_errR">
     <property name="geometry">
      <rect>
       <x>518</x>
       <y>138</y>
       <width>250</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:16px;
color: rgb(251, 173, 88);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="MyLineEdit" name="lineEdit_password1R">
     <property name="geometry">
      <rect>
       <x>518</x>
       <y>222</y>
       <width>317</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	font-size:20px;
}
QLineEdit#lineEdit_password1R{
border:1px solid rgb(211,211,211);
}
QLineEdit:hover#lineEdit_password1R{
border:1px solid rgb(255,170,0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_star4">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>333</y>
       <width>6</width>
       <height>6</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(251, 173, 88);</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="MyLineEdit" name="lineEdit_password2R">
     <property name="geometry">
      <rect>
       <x>518</x>
       <y>267</y>
       <width>317</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	font-size:20px;
}
QLineEdit#lineEdit_password2R{
border:1px solid rgb(211,211,211);
}
QLineEdit:hover#lineEdit_password2R{
border:1px solid rgb(255,170,0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_star1">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>193</y>
       <width>6</width>
       <height>6</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(251, 173, 88);</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_CloseR">
     <property name="geometry">
      <rect>
       <x>858</x>
       <y>17</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_CloseR
{
      border-image: url(:/image/Close01.png);
}
QPushButton:hover#btn_CloseR
{
      border-image: url(:/image/Close_hover.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="MyLineEdit" name="lineEdit_AccountR">
     <property name="geometry">
      <rect>
       <x>518</x>
       <y>177</y>
       <width>317</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	font-size:20px;
}
QLineEdit#lineEdit_AccountR{
border:1px solid rgb(211,211,211);
}
QLineEdit:hover#lineEdit_AccountR{
border:1px solid rgb(255,170,0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Change1">
     <property name="geometry">
      <rect>
       <x>794</x>
       <y>224</y>
       <width>40</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushbutton{

width:  16px;
height: 16px;
qproperty-flat: true;
margin-right: 4px;
border: none;
border-width: 0;
background: transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Change2">
     <property name="geometry">
      <rect>
       <x>794</x>
       <y>269</y>
       <width>40</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushbutton{
background: transparent;
width:  16px;
height: 16px;
qproperty-flat: true;
margin-right: 4px;
border: none;
border-width: 0;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>408</x>
       <y>177</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:bold &quot;Agency FB&quot;;
font-size:18px;
color: rgb(110, 110, 110);</string>
     </property>
     <property name="text">
      <string>账号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>408</x>
       <y>267</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:bold &quot;Agency FB&quot;;
font-size:18px;
color: rgb(110, 110, 110);</string>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>408</x>
       <y>312</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:bold &quot;Agency FB&quot;;
font-size:18px;
color: rgb(110, 110, 110);</string>
     </property>
     <property name="text">
      <string>人员姓名</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>408</x>
       <y>222</y>
       <width>100</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:bold &quot;Agency FB&quot;;
font-size:18px;
color: rgb(110, 110, 110);</string>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_user">
     <property name="geometry">
      <rect>
       <x>716</x>
       <y>314</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{
	font-size:20px;
}
QComboBox#combo_user{
border:1px solid rgb(211,211,211);
height: 50px;
}
QComboBox:hover#combo_user{
border:1px solid rgb(255,170,0);
}
QComboBox::down-arrow#combo_user{
background-image: url(:/image/comboBox_btn (2).png);
background-repeat:no-repeat;
background-position:center;
height:5px;
width:12px;
}
QComboBox::down-arrow:hover#combo_user{
background-image: url(:/image/comboBox_btn (1).png);
background-repeat:no-repeat;
background-position:center;
height:5px;
width:12px;
}
QComboBox::down-arrow:checked#combo_user{
background-image: url(:/image/comboBox_btn (3).png);
background-repeat:no-repeat;
background-position:center;
height:5px;
width:12px;
}
QComboBox::drop-down{
	background-color: rgb(255,255,255);
}   

QComboBox QAbstractItemView::item {
    height: 45px;   
/* 项的高度（设置pComboBox-&gt;setView(new QListView());后，该项才起作用） */
}

/* QComboBox中的垂直滚动条 */
QComboBox QAbstractScrollArea QScrollBar:vertical {
    width: 10px;
    background-color: #d0d2d4;   /* 空白区域的背景色  灰色green */
}
 
QComboBox QAbstractScrollArea QScrollBar::handle:vertical {
    border-radius: 5px;   /* 圆角 */
    background: rgb(160,160,160);   /* 小方块的背景色深灰lightblue */
}
 
QComboBox QAbstractScrollArea QScrollBar::handle:vertical:hover {
    background: rgb(90, 91, 93);   /* 越过小方块的背景色yellow */
}
</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>挂号员</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>护士管理员</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>药剂师</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>医技师</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>划价收费员</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>门诊医生</string>
      </property>
     </item>
    </widget>
    <zorder>lineEdit_Username</zorder>
    <zorder>lineEdit_password1R</zorder>
    <zorder>btn_MinR</zorder>
    <zorder>label_star3</zorder>
    <zorder>label_picLoveR</zorder>
    <zorder>label_2R</zorder>
    <zorder>label_star2</zorder>
    <zorder>btn_RegisterR</zorder>
    <zorder>label_errR</zorder>
    <zorder>label_star4</zorder>
    <zorder>lineEdit_password2R</zorder>
    <zorder>label_star1</zorder>
    <zorder>btn_CloseR</zorder>
    <zorder>lineEdit_AccountR</zorder>
    <zorder>btn_Change1</zorder>
    <zorder>btn_Change2</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label</zorder>
    <zorder>label_4</zorder>
    <zorder>combo_user</zorder>
   </widget>
   <widget class="QWidget" name="rs">
    <widget class="QPushButton" name="min_rs">
     <property name="geometry">
      <rect>
       <x>823</x>
       <y>17</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#min_rs{
border-image: url(:/image/Min01.png);
}
QPushButton:hover#min_rs{
border-image:url(:/image/Min_hover.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="close_rs">
     <property name="geometry">
      <rect>
       <x>858</x>
       <y>17</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#close_rs
{
      border-image: url(:/image/Close01.png);
}
QPushButton:hover#close_rs
{
      border-image: url(:/image/Close_hover.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_gx">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>155</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:26px;
color: rgb(115, 115, 115);</string>
     </property>
     <property name="text">
      <string> 恭喜您</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_success">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>200</y>
       <width>214</width>
       <height>54</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:52px;
color: rgb(229, 141, 62);</string>
     </property>
     <property name="text">
      <string>发送成功</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_right">
     <property name="geometry">
      <rect>
       <x>638</x>
       <y>159</y>
       <width>204</width>
       <height>204</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/image/RegisterSuccess.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_gotologin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>339</y>
       <width>237</width>
       <height>48</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#btn_gotologin{
font-size:24px;
background-color:rgb(251,173,88);
color: rgb(254, 245, 237);
border-radius:5px;
}
QPushButton:hover#btn_gotologin{
background-color:rgb(255,115,65);
color: rgb(254, 245, 237);
border-radius:5px;
}



</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_success_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>214</width>
       <height>54</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:52px;
color: rgb(229, 141, 62);</string>
     </property>
     <property name="text">
      <string>邮箱激活</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_AccountR</tabstop>
  <tabstop>lineEdit_password1R</tabstop>
  <tabstop>lineEdit_password2R</tabstop>
  <tabstop>lineEdit_Username</tabstop>
  <tabstop>btn_RegisterR</tabstop>
  <tabstop>min_rs</tabstop>
  <tabstop>close_rs</tabstop>
  <tabstop>btn_gotologin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
